src/components/Kite/story.tsx

```diff
@@ -0,0 +1,173 @@
+  const [isOpen, setIsOpen] = useState(false);
+
+  const showKite = () => {
+    setIsOpen(true);
+  };
+
+  const hideKite = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <>
+      <div>
+        <Button modifiers={['primary']} onClick={showKite}>
+          Show Kite
+        </Button>
+      </div>
+      <Kite visible={isOpen} onClose={hideKite}>
+        <Kite.Header canBeDismissed title="The Title" onClose={hideKite}>
+          <Kite.Icon
+            fill="green"
+            height="20"
+            name="check"
+            style={{ display: 'block' }}
+            width="20"
+          />
+        </Kite.Header>
+      </Kite>
+    </>
+  );
+}
+
+function ErrorKiteExample() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const showKite = () => {
+    setIsOpen(true);
+  };
+
+  const hideKite = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <>
+      <div>
+        <Button modifiers={['primary']} onClick={showKite}>
+          Error Kite
+        </Button>
+      </div>
+      <Kite visible={isOpen} onClose={hideKite} ttl={3000}>
+        <Kite.Header canBeDismissed title="Error Kite" onClose={hideKite}>
+          <Kite.Icon
+            fill="red"
+            height="20"
+            name="warning"
+            style={{ display: 'block' }}
+            width="20"
+          />
+        </Kite.Header>
+      </Kite>
+    </>
+  );
+}
+
+function EverythingKiteExample() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const showKite = () => {
+    setIsOpen(true);
+  };
+
+  const hideKite = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <>
+      <div>
+        <Button modifiers={['primary']} onClick={showKite}>
+          Everything Kite
+        </Button>
+      </div>
+      <Kite visible={isOpen} onClose={hideKite}>
+        <Kite.Header
+          canBeDismissed
+          title="Kite with content"
+          onClose={hideKite}
+        >
+          <Kite.Icon
+            fill="red"
+            height="20"
+            name="warning"
+            style={{ display: 'block' }}
+            width="20"
+          />
+        </Kite.Header>
+        <Kite.Content>
+          <div className="t-flex">
+            <div>
+              <span>The quick brown fox jumps over the lazy dog!</span>
+            </div>
+            <div className="mt-2">
+              <Button modifiers={['primary']} onClick={() => {}}>
+                Click me!
+              </Button>
+            </div>
+          </div>
+        </Kite.Content>
+      </Kite>
+    </>
+  );
+}
+
+storiesOf('Galaxies/Kite', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
+    'Overview',
+    () => (
+      <Kite visible={false} ttl={3000} onClose={() => {}}>
+        <Kite.Header canBeDismissed title="Kite Title" onClose={() => {}}>
+          <Kite.Icon
+            fill="green"
+            height="20"
+            name="check"
+            style={{ display: 'block' }}
```

Comment:
What do you think of making the icon size and `display: block` part of the component's built-in styles?
(Or are there already defaults, and this is showing the user how to make overrides?)

Reply:
I'm using the props that are already defined on Rover Icon component. There are defaults and this is showing how to make overrides.